### DISCUSSION TITLE
Is all fields mandatory?

### DISCUSSION THREAD

**Discussion by captainDuckay** [morenoh149/react-native-contacts#640] [Q&A]
I'm a big fan of typescript.

According to the contact interface, all but 1 property is mandatory.
Is this by design? If a contact has doesn't have a middlename, should that be demonstrated by an empty string, instead of types null or undefined?

I can always ignore the ignore the error, by // @ts-ignore, but it doesn't seem right?

**Comment by morenoh149** [comment]
if you verify it is not required, please submit a PR changing the optionality of the field. Thanks.
